annif/eval.py

```diff
@@ -145,10 +145,48 @@ def _evaluate_samples(self, y_true, y_pred, metrics='all'):
 
         return results
 
-    def results(self, metrics='all'):
+    def output_result_per_label(self, y_true, y_pred, results_file):
```

Comment:
I'd like to avoid talking of labels (in this sense) in the codebase since it already has a different meaning (like in SKOS) - could you rename this to `output_result_per_subject`? And the docstring as well.

Reply:
You're right, I fixed this